wls-broadcast-service/src/test/java/de/muenchen/oss/wahllokalsystem/utils/Testdaten.java

```diff
@@ -0,0 +1,30 @@
+package de.muenchen.oss.wahllokalsystem.utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.muenchen.oss.wahllokalsystem.domain.Message;
+import java.time.LocalDateTime;
+
+public class Testdaten {
+
+    public static String WAHLBEZIRK_ID = "123";
+    public static String MESSAGE = "Das ist ein Test";
+    public static LocalDateTime UHRZEIT = LocalDateTime.now();
+
+    public static Message createMessage(String wahlbezirkID, String nachricht, LocalDateTime time) {
+        Message message = new Message();
+        message.setEmpfangsZeit(time);
+        message.setNachricht(nachricht);
+        message.setWahlbezirkID(wahlbezirkID);
+        return message;
+    }
+
+    public static String asJsonString(final Object obj) {
+        try {
+            final ObjectMapper mapper = new ObjectMapper();
```

Comment:
das kann wieder zu Problemen führen wenn der Mapper nicht korrekt konfiguriert ist. In den Tests kann man sich auch den `ObjectMapper` per Dependency-Injection ziehen und an den Tests `throws Exception` werfen. Eine Wandlung der Exception im Tests ist nicht erforderlich. Kommt die ungewollte Exception geht der Test in Error.

Reply:
resolved 8405712f2b973b7c9ffaaac2a207b9d108b73790